lib/results.js

```diff
@@ -157,8 +157,10 @@ function encodeResult (res, count) {
     if (res.at) {
         output += inner + 'at: ' + res.at + '\n';
     }
-    if (res.operator === 'error' && res.actual && res.actual.stack) {
-        var lines = String(res.actual.stack).split('\n');
+    var error = (res.actual instanceof Error ? res.actual : res.error);
```

Comment:
As a test that's great as long as the test is skipped in non-node environments.

For actually fixing it, it's probably easier to just continue ducktyping the existence of a `stack` property.